completions/bash/skopeo

```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
+# shellcheck shell=bash disable=SC2207
 
-: ${PROG:=$(basename ${BASH_SOURCE})}
+PROG="skopeo"
```

Comment:
It would be cleaner to just remove the variable at this point; we rely on `PROG` being `skopeo` for the `_skopeo_skopeo` function to be called. (Or, well, we could not do that, and handle that special case cleanly as a true special case.)

(Alternatively, to just fix the warning without changing the rest, `${BASH_SOURCE[0]}` might work.)

Reply:
Done